test/e2e/util/util.go

```diff
@@ -214,12 +216,26 @@ func WaitForNReadyNodes(t *testing.T, ctx context.Context, client crclient.Clien
 			}
 		}
 		if len(readyNodes) != int(n) {
+			readyNodeCount = len(readyNodes)
 			return false, nil
 		}
 		t.Logf("All nodes are ready. Count: %v", len(nodes.Items))
 		return true, nil
 	}, ctx.Done())
-	g.Expect(err).NotTo(HaveOccurred(), "failed to ensure guest nodes became ready")
+	if err != nil {
```

Comment:
If we did this unconditionally we would implicitly test that it actually works, right now that is untested unless we fail at the right place

Reply:
updated to always write out